Gemfile.lock

```diff
@@ -224,10 +224,7 @@ GEM
     faraday-net_http (3.0.2)
     fauxhai-ng (9.3.0)
       net-ssh
-    ffi (1.16.3)
-    ffi (1.16.3-x64-mingw-ucrt)
-    ffi (1.16.3-x64-mingw32)
-    ffi (1.16.3-x86-mingw32)
+    ffi (1.17.0)
```

Comment:
Need to make sure all build channels are ok with the upgrade. It should be harmless, but `ffi` has caused nebulous problems in the past.

Reply:
agreed

Reply:
I added ffi 1.16 back into the gemfile.lock. I didn't want to rock the boat too much here.